action.yml

```diff
@@ -46,6 +46,13 @@ runs:
           curl -O "https://www.python.org/ftp/python/${Version}/${Name}.zip"
           Expand-Archive "${Name}.zip"
           echo "${PWD}\${Name}" >> $GITHUB_PATH
+
+          # Some xplat code uses `python3` because that is the executable name on Unix/macOS
+          New-Item -Path python3.exe -ItemType SymbolicLink -Value python.exe -ErrorAction Ignore
```

Comment:
Not 100% sure if we need this, since `python3` was resolving before.

Reply:
My guess is that it was resolving to the app execution alias which in some OS states is able to do something like defer to python.exe? I felt like it was safer to ensure that both exes resolve at the same part of the %Path%.